Can not build clang-ctu under VS2017 on Win7 64bit

when I build clang-ctu with llvm5.0, one of clang-ctu projects which named "clangStaticAnalyzerFrontend" with target type library cannot compile successful.

It show me:
39>------ Build started: Project: clangStaticAnalyzerFrontend, Configuration: Debug x64 ------
39>AnalysisConsumer.cpp
39>E:\svn\developer_codes\llvm-5.0.1_ctu\llvm-release_50\tools\clang\lib\StaticAnalyzer\Frontend\AnalysisConsumer.cpp(52): fatal error C1083: **Cannot open include file: 'sys/file.h'**: No such file or directory
39>Done building project "clangStaticAnalyzerFrontend.vcxproj" -- FAILED.
========== Build: 38 succeeded, 1 failed, 6 up-to-date, 0 skipped ==========

Look into the file "AnalysisConsumer.cpp", at the head of the file where specify the include file, it has two line:
#include <sys/file.h>
#include <unistd.h>

I search these two files on google, **it looks like those files are defined on "Linux" system.**
Does it means clang-ctu only can be compiled under Linux system?

I think clang-ctu should be compiled under Windows system like the "llvm-mirror/clang".
Can you fix this issue?

Thanks

--- a/lib/StaticAnalyzer/Frontend/AnalysisConsumer.cpp
+++ b/lib/StaticAnalyzer/Frontend/AnalysisConsumer.cpp
@@ -48,9 +48,6 @@
 #include <queue>
 #include <utility>
 #include <fstream>
-#include <fcntl.h>
-#include <sys/file.h>
-#include <unistd.h>
 
 using namespace clang;
 using namespace ento;
@@ -422,18 +419,6 @@ void AnalysisConsumer::storeTopLevelDecls(DeclGroupRef DG) {
 
     LocalTUDecls.push_back(*I);
   }
-}
-
-void lockedWrite(const std::string &fileName, const std::string &content) {
-  if (content.empty()) 
-    return;
-  int fd = open(fileName.c_str(), O_CREAT|O_WRONLY|O_APPEND, 0666);
-  flock(fd, LOCK_EX);
-  ssize_t written = write(fd, content.c_str(), content.length());
-  assert(written == content.length());
-  (void)written;
-  flock(fd, LOCK_UN);
-  close(fd);
 }
 
 static bool shouldSkipFunction(const Decl *D,

